### DISCUSSION TITLE
add HighestAwardKind and HighestAwardDate to API_GetGameInfoAndUserProgress

### DISCUSSION THREAD

**Discussion by krupupakku** [RetroAchievements/RAWeb#2381] [Ideas]
As the title suggests, 
I believe it would make sense to also add HighestAwardKind and HighestAwardDate when calling the game info and user progress. 

The HighestAwardKind provides information on whether a game has been beaten (mastered could be calculated though earned achievements). 

I think this would make API_GetGameInfoAndUserProgress the definitive call if you want to know your progress on a specific game.

Thank you!

**Comment by wescopeland** [comment]
Implemented in https://github.com/RetroAchievements/RAWeb/pull/2382.

For future API enhancement requests, please direct them to the dedicated thread [here](https://github.com/RetroAchievements/RAWeb/discussions/2081). Thank you 🙏

**Reply by krupupakku** [reply]
Hey thank you for the quick implementation. for future ideas I'll use the suggested thread. thanks!
